contentcuration/contentcuration/utils/parser.py

```diff
@@ -102,3 +103,17 @@ def parse_exponent(text):
 
 def to_en(text):
     return text.replace(SEP, '').replace(POINT, '.')
+
+
+def load_json_string(json_string):
+    """
+        Using code from https://grimhacker.com/2016/04/24/loading-dirty-json-with-python/
+        Instead of using ast.literal_eval to process malformed json, load this way
+        Arg: json_string (str) to parse
+        Returns json generated from string
+    """
+    regex_replace = [(r"([ \{,:\[])(u)?'([^']+)'", r'\1"\3"'), (r" False([, \}\]])", r' false\1'), (r" True([, \}\]])", r' true\1')]
```

Comment:
This feels a little opaque to me, as necessary as it may be!  Any thoughts on how to make it more trustworthy? Tests?

Reply:
Yep! Added a test here: https://github.com/learningequality/studio/pull/1016/files#diff-a5c404d5320b9060cedcea4918c2d6f4R42